api/tests/unit/domain/usecases/find-paginated-campaign-managments_test .js

```diff
@@ -0,0 +1,25 @@
+const { expect, sinon } = require('../../../test-helper');
+
+const findPaginatedCampaignManagements = require('../../../../lib/domain/usecases/find-paginated-campaign-managements');
+
+describe('Unit | Domain | Use Cases | find-paginated-campaign-managments', () => {
+
+  describe('#findPaginatedCampaignManagement', () => {
+
+    it('should return the campaigns of the given organization', () => {
+      const campaignManagementRepository = { findPaginatedCampaignManagements: sinon.stub() };
+
+      // given
+      const organizationId = 251;
+      const expectedPromise = Symbol('campaign managment');
+      const page = { number: 1, size: 3 };
+      campaignManagementRepository.findPaginatedCampaignManagements.withArgs({ organizationId, page }).returns(expectedPromise);
+
+      // when
+      const promise = findPaginatedCampaignManagements({ organizationId, page, campaignManagementRepository });
+
+      // then
+      expect(promise).to.equals(expectedPromise);
```

Comment:
et ici : `expect(results).to.equals(expectedResults);`

Reply:
done